.github/workflows/manpages.yml

```diff
@@ -33,18 +34,26 @@ jobs:
             return el
           end
           EOF
-          find $MAN_DIR -iname '*.md' -type f -exec sh -c 'relative_path=$(realpath --relative-to=$MAN_DIR $0) && stripped_path=${relative_path#man} && pandoc -f gfm -t html5 -s --lua-filter=link-fixup.lua -o output/${stripped_path%.md}.html ${0}' {} \;
+          find $MAN_DIR -iname '*.md' -type f -exec sh -c '\
+            relative_path="$(realpath --relative-to=$MAN_DIR $0)" \
+            && stripped_path="${relative_path#man}" \
+            && section="${stripped_path%%/*}" \
+            && file_name="${stripped_path#*/}" \
+            && name="${filename%.md}" \
```

Comment:
Do you mean file_name?
(Otherwise LGTM, after this is fixed ill run a test run of this on my test site and report back :)

Reply:
[Filename](https://dictionary.cambridge.org/dictionary/english/filename) is one word :)

Reply:
Fixed :)